config/settings.yml

```diff
@@ -97,11 +97,11 @@ maintenance_mode_enabled?: <%= ENV.fetch('MAINTENANCE_MODE', nil)&.downcase&.eql
 feature_flags_enabled?: true
 active_features: []
 
-reject_refuse_messaging_released_at: <%= DateTime.new(2018, 4, 24, 23, 05, 0).change(offset: '+1:00') %>
+reject_refuse_messaging_released_at: 2018-04-24
```

Comment:
Is the time significant here?

Also, is it possible that we can dispense with the `reject_refuse_messaging_released_at` method in `lib/release.rb` after this change? It looks as though it is just catering for the ambiguous type of that setting. I suspect that this fix exists in a lot of places.

Also also, is the `reject_refuse_messaging_released?` method in that same class redundant now, as we are a long way past the release date?

Reply:
All good questions! I was just looking at that one again now - I'm going to add the time back in for safety, although I'm not sure it's entirely relevant at this point. There's also some tidying up of tests and other code that can be done, so I'll add an extra commit with that.